code/game/turfs/turf.dm

```diff
@@ -420,9 +420,12 @@
 		if(istype(src, /turf/simulated))
 			var/turf/simulated/T = src
 			T.dirt = 0
+			T.color = null
 		for(var/obj/effect/O in src)
-			if(istype(O,/obj/effect/rune) || istype(O,/obj/effect/decal/cleanable) || istype(O,/obj/effect/overlay))
+			if(istype(O,/obj/effect/decal/cleanable) || istype(O,/obj/effect/overlay))
```

Comment:
As demonstrated to you, this is untrue. All crayon objects use the path /obj/effect/decal/cleanable/crayon, which is handled by the Istype(O,/obj/effect/decal/cleanable) check. I have tested this extensively in game, further, and can verify to you that crayon graffiti is 100% without a doubt nor exception cleanable.